doc/z-chap9.xml

```diff
@@ -3,7 +3,9 @@
   <Section><Heading>Visualising a digraph</Heading>
     <#Include Label="Splash">
     <#Include Label="DotDigraph">
+    <#Include Label="DotColoredDigraph">
```

Comment:
This line isn't necessary, and is probably what is causing the CI to fail, I suggest removing it.